templates/service-bus-sessions.json

```diff
@@ -1,222 +1,514 @@
+              "limit": {
+                "timeout": "PT1H"
+              },
+              "runAfter": {
+                "Complete_initial_message_in_queue": [
+                  "Succeeded"
+                ]
+              },
+              "type": "Until"
             }
-        },
-        "apiSummaries": [
-            {
-                "type": "apiconnection",
-                "displayName": "${Resources.SERVICEBUS}",
-                "iconUri": "https://connectoricons-prod.azureedge.net/servicebus/icon_1.0.1019.1195.png",
-                "brandColor": "#59B4D9"
+          },
+          "description": "Follow the Try/Catch pattern using Scopes to process messages in a Try block but appropriately handle errors with a Catch block",
+          "runAfter": {
+            "Init_isDone": [
+              "Succeeded"
+            ]
+          },
+          "type": "Scope"
+        }
+      },
+      "contentVersion": "1.0.0.0",
+      "outputs": {},
+      "parameters": {
+        "$connections": {
+          "defaultValue": {},
+          "type": "Object"
+        }
+      },
+      "triggers": {
+        "When_a_message_is_received_in_a_queue_(peek-lock)": {
+          "inputs": {
+            "host": {
+              "connection": {
+                "name": "@parameters('$connections')['servicebus']['connectionId']"
+              }
             },
-            {
-                "type": "apiconnection",
-                "displayName": "${Resources.SERVICEBUS}",
-                "iconUri": "https://connectoricons-prod.azureedge.net/servicebus/icon_1.0.1019.1195.png",
-                "brandColor": "#59B4D9"
+            "method": "get",
+            "path": "/@{encodeURIComponent('')}/messages/head/peek",
+            "queries": {
+              "queueType": "Main",
+              "sessionId": "Next Available"
             }
-        ],
-        "changedTime": "2017-07-05T00:00:59.000Z",
-        "createdTime": "2017-04-04T00:00:59.000Z",
-        "popularity": 3
-    }
+          },
+          "recurrence": {
+            "frequency": "Minute",
+            "interval": 3
+          },
+          "type": "ApiConnection"
+        }
+      }
+    },
+    "connectionReferences": {
+      "servicebus": {
+        "connection": {
+          "id": ""
+        },
+        "api": {
+          "id": "/subscriptions/{0}/providers/Microsoft.Web/locations/{1}/managedApis/servicebus"
+        }
+      }
+    },
+    "apiSummaries": [
+      {
+        "type": "apiconnection",
+        "displayName": "${Resources.SERVICEBUS}",
+        "iconUri": "https://connectoricons-prod.azureedge.net/servicebus/icon_1.0.1019.1195.png",
+        "brandColor": "#59B4D9"
+      },
+      {
+        "type": "apiconnection",
+        "displayName": "${Resources.SERVICEBUS}",
+        "iconUri": "https://connectoricons-prod.azureedge.net/servicebus/icon_1.0.1019.1195.png",
+        "brandColor": "#59B4D9"
+      }
+    ],
+    "changedTime": "2017-07-05T00:00:59.000Z",
```

Comment:
Looks good to me.  Just update `changedTime` here.

Reply:
done